File: src/SimpleStocker.Web/src/app/models/client.model.ts
import { BaseModel } from './basemodel.model';

export class Client extends BaseModel {
  name: string = '';
  email: string = '';
  phoneNumber: string = '';
  address: string = '';
  addressNumber: string = '';
  active: boolean = true;
  birthDate: Date = new Date();

  constructor() {
    super();
    this.name = '';
    this.email = '';
    this.phoneNumber = '';
    this.address = '';
    this.addressNumber = '';
    this.active = false;
    this.birthDate = new Date();
  }
}

File: src/SimpleStocker.ClientApi/Models/ClientModel.cs
namespace SimpleStocker.ClientApi.Models
{
    public class ClientModel
    {
        public long Id { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime UpdatedDate { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string PhoneNumer { get; set; } = string.Empty;
        public string Address { get; set; } = string.Empty;
        public string AddressNumber { get; set; } = string.Empty;
        public bool Active { get; set; }
        public DateTime BirthDate { get; set; }
    }
}
